admin-manual/maintenance/common-atom-queries.rst

```diff
@@ -0,0 +1,2000 @@
+"digital image" throughout our descriptions. Now we want to replace all 
+occurences of "digtal image" in the *Extent and medium* field across all our
+descriptions in English with the corrected version, "digital image":
+
+.. code-block:: mysql
+
+   UPDATE information_object_i18n SET extent_and_medium=REPLACE(extent_and_medium, 'digtal image', 'digital image') WHERE culture='en';
+
+Another example: let's globally replace the word "tea" with the word "coffee" in the
+scope and content field of all English descriptions across AtoM:
+
+.. code-block:: mysql
+
+   UPDATE information_object_i18n SET scope_and_content=REPLACE(scope_and_content, 'tea', 'coffee') WHERE culture='en';
+
+If we wanted to limit this change to a specific field in a specific 
+description, we can include a join on the slug table, and then use the 
+:term:`slug` of the target description as a parameter:
+
+.. code-block:: mysql
+
+   UPDATE information_object_i18n io JOIN slug s ON s.object_id=io.id SET io.scope_and_content=REPLACE(scope_and_content, 'tea', 'coffee') WHERE io.culture='en'AND s.slug="my-new-slug";
+
+Replace ``my-new-slug`` in the above example with the slug of your target 
+description.
+
+**Globally updating a field**
+
+Some of the fields in AtoM's Description control :term:`area <information area>` 
+might have the same information across all descriptions. For example, if you 
+wanted to add the same data to every "Rules and conventions used" field in your
+descriptions, we could use a query like so: 
+
+.. code-block:: mysql
+
+   UPDATE information_object_i18n io18 
+   LEFT JOIN information_object io ON io18.id=io.id 
+   SET io18.rules="ISAD(G): General International Standard Archival Description, Second Edition, Ottawa, 2000."
+   WHERE io18.culture='en' AND io.parent_id IS NOT NULL;
+
+We could also add another parameter and limit this to top-level descriptions
+associated with a particular :term:`repository`. In the following example,
+only the top-level descriptions will be affected, since they are the only ones
+that will have a repository ID associated directly with them (AtoM uses
+inheritance to associate the lower level descriptions with the repository, but
+these descriptions won't have the associated repository ID stored directly in 
+the database).
+
+To do so, we will first need to know the object ID of the repository record - 
+see above: 
+
+* :ref:`cli-object-id`
+
+Once we know the related repository's ID, we can modify the above example by
+adding the repository ID as an additional criteria. In this example, our 
+repository's object ID was 3169:
+
+.. code-block:: mysql
+
+   UPDATE information_object_i18n io18 
+   LEFT JOIN information_object io ON io18.id=io.id 
+   SET io18.rules="ISAD(G): General International Standard Archival Description, Second Edition, Ottawa, 2000."
+   WHERE io18.culture='en' 
+   AND io.repository_id='3169'
+   AND io.parent_id IS NOT NULL;
```

Comment:
Same about the last condition and same about the descendants.

Reply:
in this one i already had a note in place above mentioning that this will only affect the top-level descriptions, so i think it's okay. For the other example, it's  not really useful to update the pub status of just the parents, so i'm hoping to find a query that will work for descendants as well.